ts/features/itwallet/presentation/screens/ItwPresentationCredentialDetailScreen.tsx

```diff
@@ -87,19 +106,25 @@ const getCtaProps = (
 ): CredentialCtaProps | undefined => {
   const { parsedCredential } = credential;
 
+  const onPress = () => {
+    if (credential.credential === "ITW_TS_V2") {
```

Comment:
`credential.credential` is the credential's JWT. I don't think it can ever be `ITW_TS_V2`. Is it a mistake? Maybe we want to check whether this is the health insurance card?